### ISSUE TITLE
quick win: is it possible to support handlebars-helpers?

### ISSUE THREAD

**Issue by jpinho** [johnknoop/vscode-handlebars-preview#59]
would be great if we can support these helpers on the plugin 🙏  https://github.com/helpers/handlebars-helpers

**Comment by jpinho** [comment]
Success 🚀 !

I was able to sort it. So I've a project with a couple of Lambdas in NodeJS. And my confusion was, I use typescript and register my on helpers, but those did seem to be picked up by the extension.

So I read the code and the extension looks up for something.handlebars.js (hbs.js would also work). And bcoz I didn't wanted an "helper" file to be bundled into my lambda code, at the root of the workspace I added the 2 modules to the global package.json:

- handlerbars
- handlerbars-helpers

And then created a file with this content:

```js
/* eslint-disable @typescript-eslint/no-var-requires */

module.exports = require('handlebars-helpers')([
  'math',
  'number',
  'date',
  'comparison',
  'match',
  'array',
  'regex',
  'collection',
  'object',
  'html',
  'markdown',
  'string',
  'url',
  'moment',
]);
```

Note, initially I had the first line with an import and it didn't work. But `require` does the job 🙏 

I love this extension so much! ❤️
